defender/README.md

```diff
@@ -49,4 +49,51 @@ the Autotasks to verify that the Autotasks are performing as expected.
 
```

Comment:
Should we remove the above sections from lines 28-39? 
* `## Pushing an Existing Configuration to Defender`
* `## Pulling Down a Defender Configuration`

They don't seem applicable when using serverless deployments

Reply:
TODO: Update README

Reply:
Addressed in b275551eee16572cc5286da92eadfc4240c2e27b